model/task/task.go

```diff
@@ -1370,13 +1370,28 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 }
 
+// This should reflect the addDisplayStatus aggregation step
```

Comment:
Nit: can you use the function name instead of This so it's official documentation?